external-images.yaml

```
@@ -0,0 +1,16 @@
+images:
+  - source: "alpine@sha256:0a4eaa0eecf5f8c050e5bba433f58c052be7587ee8af3e8b3910ef9ab5fbe9f5"
+    tag: "3.20.2"
+  - source: "curlimages/curl:7.78.0"
+  - source: "cypress/included:9.5.0" # 10.3.0+, latest is 12.0.1
+    amd64Only: true
+  - source: "goreleaser/goreleaser:v1.11.5"
+  # Golang image is pinned because it is force updated in the upstream
+  - source: "golang@sha256:d0b31558e6b3e4cc59f6011d79905835108c919143ebecc58f35965bf79948f4"
+    tag: "1.23.0-alpine3.20"
+    amd64Only: true
```

```suggestion
  - source: "golang@sha256:ac67716dd016429be8d4c2c53a248d7bcdf06d34127d3dc451bda6aa5a87bc06"
    tag: "1.23.1-alpine3.20"
```

amd64only flag was deleted on the main and the 1.23.1 is alredy synced